src/Features/RemotePaymentMethods/DefaultPaymentGateways.php

```diff
@@ -125,9 +126,36 @@ public static function get_all() {
 					self::get_rules_for_cbd( false ),
 				),
 			),
+			array(
+				'key'         => 'woocommerce_payments',
+				'title'       => __( 'WooCommerce Payments', 'woocommerce-admin' ),
+				'content'     => __(
+					'Manage transactions without leaving your WordPress Dashboard. Only with WooCommerce Payments.',
+					'woocommerce-admin'
+				),
+				'image'       => plugins_url( 'images/onboarding/wcpay.svg', WC_ADMIN_PLUGIN_FILE ),
+				'plugins'     => array( 'woocommerce-payments' ),
+				'description' => 'Try the new way to get paid. Securely accept credit and debit cards on your site. Manage transactions without leaving your WordPress dashboard. Only with WooCommerce Payments.',
+				'is_visible'  => array(
+					self::get_rules_for_cbd( false ),
+					self::get_rules_for_countries( self::get_wcpay_countries() ),
+				),
+			),
 		);
 	}
 
+	/**
+	 * Get array of countries supported by WCPay depending on feature flag.
+	 *
+	 * @return array Array of countries.
+	 */
+	public static function get_wcpay_countries() {
```

Comment:
It feels like this belongs somewhere else, but I can't find the right place.  Not sure if we should throw in a feature class (just for this?) or keep it here.

Reply:
Yeah, I felt somewhat the same, but it's such a small bit of logic I leaned into just keeping in local since it's relevant. Totally receptive to relocating though, if others disagree.

Reply:
Sounds good.  I'm fine leaving this in here for now until we have more logic or a separate class to move this to.